xml/planning-architecture-alternative-rhel_compute_model.xml

```diff
@@ -6,6 +6,154 @@
+     packages present in both repos are preferably installed from SLES repo.
+    </para>
+
+     <screen>zypper addrepo --name centos_c8_ardana --priority 9999 https://download.opensuse.org/repositories/systemsmanagement:/Ardana:/8:/CentOS/CentOS_7.3/centos_c8_ardana</screen>
+
+     <para>venv RPM packages can be specifically installed from OBS CentOS
+     repository:</para>
+
+     <screen>zypper install --from centos_c8_ardana venv-openstack-nova-rhel-x86_64</screen>
+   </step>
+   <step>
+    <para>
+     Next, you need to run the following command on the deployer to add the venv to package indexes:
+    </para>
+    <screen>/usr/bin/create_index --dir=/opt/ardana_packager/ardana-8/rhel_venv/x86_64</screen>
+   </step>
+   <step>
+    <para>
+     On &rhel; &compnode;s, configure OBS CentOS repository to download
+     specific RPM packages required by cloud tooling, &ostack; &o_comp; and
+     &o_netw; &comp; components. Following RPM packages are required from OBS
+     CentOS repository on the &compnode;:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       python-ardana-packager
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       openvswitch
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       python-PyYAML
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       python-Beaver
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       openstack-freezer
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       openstack-freezer-api
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       openstack-freezer-agent
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       openstack-freezer-scheduler
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       python-python-logstash
+      </para>
+     </listitem>
+    </itemizedlist>
+   </step>
+   <step>
+    <para>
+     Installing &ostack; Pike release &o_comp; &comp; version requires qemu-kvm
+     version 2.1.0 or higher. It is recommended to install the required
+     packages via &rhel; subscription to ensure continuous
+     updates. Alternatively, these packages can be installed manually from
+     http://vault.centos.org/7.3.1611/virt/x86_64/ This includes the following
+     packages:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       qemu-img-ev-2.6.0-28.el7.10.1.x86_64.rpm
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       qemu-kvm-common-ev-2.6.0-28.el7.10.1.x86_64.rpm
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       qemu-kvm-ev-2.6.0-28.el7.10.1.x86_64.rpm
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       qemu-kvm-tools-ev-2.6.0-28.el7.10.1.x86_64.rpm
+      </para>
+     </listitem>
+    </itemizedlist>
+   </step>
+  </procedure>
+  <para>
+   Below are alternative configurations for &rhel; &compnode;s.
```

Comment:
Maybe use a : at the end? That would help the stylesheets keep the para + following list together too...